src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5761,18 +5808,32 @@ static void addSlur(const Notation& notation, SlurStack& slurs, ChordRest* cr, c
             if (cr->isGrace()) {
                 newSlur->setAnchor(Spanner::Anchor::CHORD);
             }
-            if (lineType == "dotted") {
-                newSlur->setStyleType(SlurStyleType::Dotted);
-            } else if (lineType == "dashed") {
+            const auto lineType = notation.attribute("line-type");
+            if (lineType == "dashed") {
                 newSlur->setStyleType(SlurStyleType::Dashed);
+            } else if (lineType == "dotted") {
+                newSlur->setStyleType(SlurStyleType::Dotted);
+            } else if (lineType == "solid" || lineType == "") {
+                newSlur->setStyleType(SlurStyleType::Solid);
             }
             newSlur->setTick(Fraction::fromTicks(tick));
             newSlur->setStartElement(cr);
-            const auto pl = notation.attribute("placement");
-            if (pl == "above") {
-                newSlur->setSlurDirection(DirectionV::UP);
-            } else if (pl == "below") {
-                newSlur->setSlurDirection(DirectionV::DOWN);
+            const QColor color { notation.attribute("color") };
+            if (color.isValid()) {
+                newSlur->setColor(color);
```

Comment:
I believe adding `newSlur->setPropertyFlags(engraving::Pid::COLOR, engraving::PropertyFlags::UNSTYLED);` here would fix that strange partial import

Reply:
No, it doesn't work there either. The color gets exported but not shown. It's a different MuseScore error.

Reply:
Ah!

Reply:
Hmm, it does work, after import, save, close, reopen

Reply:
Weird.